exporters/otlp/common/src/main/java/io/opentelemetry/exporter/internal/grpc/UpstreamGrpcExporter.java

```diff
@@ -121,9 +117,6 @@ public void onFailure(Throwable t) {
 
   @Override
   public CompletableResultCode shutdown() {
-    if (managedChannel.isTerminated()) {
-      return CompletableResultCode.ofSuccess();
-    }
-    return ManagedChannelUtil.shutdownChannel(managedChannel);
```

Comment:
This was a bit fishy previously - we were closing user-provided channels which likely shouldn't have been happening. Now since this class is only for user-provided channels, we can simply never close it